examples/demo13.py

```diff
@@ -106,9 +106,9 @@ def main(argv):
     # something larger (but it should be <=100, the number of galaxies in this small example
     # catalog).  With 4000 galaxies in a 4k x 4k image with the WFIRST pixel scale, the effective
     # galaxy number density is 74/arcmin^2.  This is not the number density that is expected for a
-    # sample that is so bright (I<23.5) but it makes the image more visually interesting.  One could
+    # sample that is so bright (I<25.2) but it makes the image more visually interesting.  One could
```

Comment:
This was 23.5 initially, and I changed 23.5 to 25.2 everywhere. The number density will have to change and probably the comment itself.